ui/js/controllers/filters.js

```diff
@@ -51,12 +50,12 @@ treeherderApp.controller('JobFilterCtrl', [
         };
 
         $scope.isFilterOn = function (filter) {
-            if (_.keys($scope.filterGroups).indexOf(filter) !== -1) {
-                // this is a filter grouping, so toggle all on/off
-                return _.some(
-                    _.at($scope.resultStatusFilters,
-                    $scope.filterGroups[filter].resultStatuses)
-                );
+            if (Object.keys($scope.filterGroups).indexOf(filter) !== -1) {
+                const filterOn = [];
```

Comment:
It seems like we could use a ``reduce`` insead of this ``for .. of`` statement.  Would that work?